Production/govuk_ios/ViewModels/Topics/AllTopicsViewModel.swift

```diff
@@ -0,0 +1,27 @@
+import UIKit
+import Foundation
+
+class AllTopicsViewModel {
+    let analyticsService: AnalyticsServiceInterface
+    let topicAction: ((Topic) -> Void)?
+    let topics: [Topic]
+    var downloadError: TopicsListError?
```

Comment:
Do all of these need to be internal? `private` or `private(set)` for some maybe?

Reply:
They've changed a tad but have updated according to your comment